templates/vars/vnet/prod.json

```diff
@@ -51,6 +51,28 @@
           "service": "Microsoft.Storage"
         }
       ]
-    }
+    },
+    "bastionSubnetPrefix": {
+      "value": "10.48.0.0/27"
+    },
+    "paloaltoIPNextHop": {
+      "value": "10.11.8.36"
+    },
+    "routetableRoutes": {
```

Comment:
I thought prod was being left out of this PR?

Reply:
ah i see you're merging once prod is done

Reply:
Going to leave Prod out of this for now, its not due to be applied until 12th April